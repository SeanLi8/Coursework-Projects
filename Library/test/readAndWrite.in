save collection test/combined.csv
list all
add book books.csv 999999 
add book yes.csv 111111
list all
add book books.csv 111111
list all
add book books.csv 111111
list all
add book books.csv 111134
list all
add collection yes.csv
add collection books.csv
list all
save collection test/booksDupe.csv
add collection books2.csv
list all
save collection test/combined.csv
exit