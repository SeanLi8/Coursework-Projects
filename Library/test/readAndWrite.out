user: No books in system.

user: No books in system.

user: No such book in file.

user: No such file.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: To Kill a Mockingbird (Harper Lee)

user: Book already exists in system.

user: To Kill a Mockingbird (Harper Lee)

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)

user: No such collection.

user: 24 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)

user: Success.

user: 8 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka)
Kaichou wa Maid-sama! (Fujiwara Hiro)
Bakuman (Obata Takeshi)
Shingeki no Kyojin (Isayama Hajime)
Hunter X Hunter (Togashi Yoshihiro)
Shokugeki no Somua (Saeki Shun)
Haikyuu!! (Furudate Haruichi)
Slam Dunk (Inoue Takehik0)

user: Success.

user: Ending Library process.
